doc/todo/API.md

```diff
@@ -0,0 +1,13 @@
+On potential changes (including extensions) of the API/ABI please check following list:
+
+- [ ] [use case](/doc/usecases) for API exists
+- [ ] [decision](/doc/decision) for API is in step `implemented`
```

Comment:
Well the correct step depends on when this checklist must be checked. I'd say it makes more sense to say:

```suggestion
- [ ] **Before** starting to implement the change: [decision](/doc/decision) for API is in step `decided`
- [ ] **After** implementing the change: [decision](/doc/decision) for API is in step `implemented`
```